conf/services/hmrc-developer-hub.yml

```diff
@@ -5,7 +5,7 @@ serviceDescription: |
 serviceDomain: developer.service.hmrc.gov.uk
 serviceUrl: /api-documentation
 contactFrontendServiceId: hmrc-developer-hub
-complianceStatus: partial
+complianceStatus: noncompliant
```

Comment:
Have you reached out to someone from the developer hub team about this chagne?

Reply:
Yes @adampridmore has approved.